lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -176,7 +180,7 @@ public void execute(Publisher publisher) throws ConnectorException {
       ors.close();

```

Comment:
I realize this isn't new code but it feels like there should be a TODO/task to clean up the way that the result sets are closed so that they are performed in a finally block or opened in a try-with-resources.

Reply:
TODO added.